_layouts/significant-change-pilot.html

```diff
@@ -67,145 +67,218 @@ <h4 class="mobile-nav-title">You can also browse the topics below.</h4>
 
 		<div id="overview" class="">
 			<h2>Overview</h2>
-			<p><b>Pilot Participants:</b> CSPs</p>
-			<p><b>Participant Target:</b> 20</p>
-			<p><b>Application Period:</b> July 8, 2024 - July 26, 2024</p>			
-			<p><b>Selection Period:</b> July 26, 2024 - August 16, 2024</p>			
-			<p><b>Pilot Kickoffs:</b> September 1, 2024 - September 15, 2024</p>
-			<p><b>Pilot Period:</b> September 15, 2024 - December 31, 2024</p>
-			<p><b>Pilot Closeout Date:</b> January 31, 2025 </p>
 			
 			<h3 id="purpose">Purpose</h3>
-			<p>In FedRAMP’s agile delivery pilot, cloud service providers (CSPs) will be able to launch new services and features without agency approval. The data gathered during this pilot will help inform program-wide changes to streamline the current process for change management. Our long-term goal is to shift the FedRAMP process to one that is based on continuous assessment rather than point-in time snapshots. </p>
+			<p>The FedRAMP agile delivery pilot allows cloud service offerings (CSOs) to launch new services and features without requiring agency pre-approval. The data gathered during this pilot will help inform program-wide updates to streamline the current process for change management. Our long-term goal is to shift the FedRAMP process to one that is based on continuous assessment rather than point-in time snapshots. </p>
 
-			<p>Currently, CSP must go through independent testing and approval before rolling out security-relevant changes such as launching a new feature. While this process helps ensure security impacts are understood in advance, it adds unpredictability and delay to the act of product improvement, creating risks and opportunity costs, and delays in agency access to new features.</p>
+			<p>Currently, CSOs must go through independent testing and must receive agency pre-approval before rolling out significant changes such as launching a new feature. While this process helps ensure security impacts are understood by stakeholders in advance, it adds unpredictability and delay to the act of product improvement, creating risks and opportunity costs, and impedes agency access to new features.</p>
```

Comment:
Minor, non-blocking: I would think there is a pause before such, add a comma?

```suggestion
			<p>Currently, CSOs must go through independent testing and must receive agency pre-approval before rolling out significant changes, such as launching a new feature. While this process helps ensure security impacts are understood by stakeholders in advance, it adds unpredictability and delay to the act of product improvement, creating risks and opportunity costs, and impedes agency access to new features.</p>
```